lefthook.yml

```diff
@@ -17,3 +17,10 @@ pre-commit:
       files: git diff --name-only --staged
       glob: "*.rb"
       run: bin/dirty-rubocop --uncommitted --force-exclusion {files}
+
+post-merge:
+  commands:
+    setup-dev:
+      files: git diff --name-only HEAD@\{1\} HEAD
+      glob: '{Gemfile.lock,frontend/package-lock.json,db/migrate/*.rb,modules/*/db/migrate/*.rb}'
+      run: bin/setup_dev
```

Comment:
That won't work of course for devs with docker setup. So, please, announce it, once it is merged, so everyone can adapt their `lefthook-local.yml`

Reply:
you know by chance out of your head, you you could switch off a lefthook hook inside the local override? If I do not want to run the setup?